II.pass-tests/09-car-sales/car-sales.js

```diff
@@ -0,0 +1,29 @@
+ function sale(carsSold) {
```

Comment:
what if we add other car makers to our test?
Think of what would happen if our test had 100 cars.
There should be a way to make `sale` work with any number of cars, without specifying them in the code